### PR TITLE
Create .appveyor-build-tags

### PR DESCRIPTION
**Pull request by omichel**
This config should be used by the webots-tags appveyor project.
It is similar to .appveyor-build-tags, except for the first line.

### CODE REVIEW

**Comment by omichel** [comment]
I already updated the "webots-tag" appveyor project accordingly to use this YML file.
👍

File: .appveyor-build.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# This file is used by the "webots-tag" appveyor project, it is identical to the .appveyor-build-tag.yml file,
```
Comment (DavidMansolino) [inline .appveyor-build.yml]:
```suggestion
# This file is used by the "webots-nightly-develop" and "webots-nightly-revision" appveyor projects, it is identical to the .appveyor-build-tag.yml file,
```

**Comment by DavidMansolino** [comment]
Is it normal that the `Do not build on "Push" events` settings is enabled in the project?

**Comment by omichel** [comment]
No. I fixed that. Thanks for pointing this out.

**Pull request merged**